app/src/ui/lib/sandboxed-markdown.tsx

```diff
@@ -44,6 +44,36 @@ export class SandboxedMarkdown extends React.PureComponent<
 > {
   private frameRef: HTMLIFrameElement | null = null
   private frameContainingDivRef: HTMLDivElement | null = null
+  private contentDivRef: HTMLDivElement | null = null
+
+  /**
+   * Resize observer used for tracking height changes in the markdown
+   * content and update the size of the iframe container.
+   */
+  private readonly resizeObserver: ResizeObserver
+  private resizeDebounceId: number | null = null
+
+  public constructor(props: ISandboxedMarkdownProps) {
+    super(props)
+
+    this.resizeObserver = new ResizeObserver(this.scheduleResizeEvent)
```

Comment:
Just a drive-by comment noting that it doesn't look like we `.disconnect` or `.unobserve` on this ResizeObserver. We should at the very least call `.disconnect` when this component unmounts. We could also `.unobserve(contentDivRef)` in `setupContentDivRef` if `contentDivRef` isn't null prior to assigning it.

Reply:
Done in [6a967a7](https://github.com/desktop/desktop/pull/14196/commits/6a967a7af74b45947cf0a379ec217bd2d937c73e), thanks!!